internal/services/controller/controller.go

```diff
@@ -486,3 +518,15 @@ func isResourceAvailable(kind reflect.Type, apiResourceList *metav1.APIResourceL
 	}
 	return false
 }
+
+func applyInformer(groupVersion string, informerFactory informers.SharedInformerFactory) cache.SharedIndexInformer {
+	switch groupVersion {
+	case "storage.k8s.io/v1":
```

Comment:
Can we avoid duplicating the informer types? The informers are already defined at the top, we should consolidate the configuration there.

Reply:
This was the part which was causing problems, can only call them them the checks pass, because it tries to sync them

Reply:
Not very pretty solution :/